.github/workflows/r-conda.yml

```diff
@@ -38,8 +38,8 @@ jobs:
       - run: |
           conda init bash
           conda env list
-
-      # Runs a set of commands using the runners shell
+      - name: Fetch remote test data
+        run: wget -P xmsannotator/tests/testthat/test-data -i xmsannotator/tests/remote-files/fetchthis.txt
```

Comment:
These files are actually not needed for the tests run in the CI at the moment, but will be once we also enable the CI on the master branch.